packages/core/hooks/useOpenWithMenuItems/index.tsx

```diff
@@ -240,7 +270,61 @@ export default (fileDetails?: FileDetail, filters?: FileFilter[]): IContextualMe
     const annotationNameToAnnotationMap = useSelector(
         metadata.selectors.getAnnotationNameToAnnotationMap
     );
+    const annotations = useSelector(metadata.selectors.getAnnotations);
     const loadBalancerBaseUrl = useSelector(interaction.selectors.getLoadBalancerBaseUrl);
+    const fileService = useSelector(interaction.selectors.getFileService);
+
+    const [hasRemoteServer, uploadCsv] = useRemoteFileUpload();
+    const openInCfeCallback = React.useCallback(async () => {
```

Comment:
Did my best to match error handling elsewhere in the app, but I'd love any feedback here.